config/tips.txt

```diff
@@ -0,0 +1,136 @@
+It is considered poor taste to decapitate someone and keep their neural lace, always give people a chance to come back into the round in some way.
+Straightjackets and Electropacks should never be used without admin approval, as they are impossible to get out of.
+Stowaways have a chance to be spawnable after the round starts, however as a Stowaway you won't be defined on the crew manifest or have any ID.
+Most ship AI will start with the manifest lawset, making it follow orders of crew on the ship manifest, as defined by the Captain.
+Ionispheric anomalies may cause AI's to gain new laws, making them act in unusual or fun manners.
+You can use Intellicards to card AI's into a portable state, allowing them to be repaired and have their lawsets visible.
+Rule Lawyering as an AI or Borg with lawsets or custom lawsets defined by antagonists is not fun for anyone.
+Rule Lawering as a imprint implanted person is not fun for anyone, especially the antagonist who used it on you.
+When you are cloned, you forget the last 5 minutes of the events leading up to your death.
+Find a bug, want to give feedback or a feature suggestion? Go to our Github Page - https://github.com/UristMcStation/UristMcStation/issues and write it up!
+The best antagonist rounds usually tend to include most players in someway.
+Losing is !FUN!, each round lost is a lesson to learn and apply for the next round.
+Changelings can take peoples voices, appearance and memories.
+Bluespace Revenants will slowly distort overtime without creating runes and feeding on blood.
+Fire Extingushers can be used to wet floors to slip people with.
+You can remove fingerprints on objects by using a rag on them.
+You may leave partial fingerprints with specific types of gloves.
+You may leave fibers of identifiable clothing on any object you interact with, swapping to less identifable clothing is always wise before doing something bad.
+Flashers can repeatidly stun cyborgs and disable their movement and tools.
+Emagging a cyborg only unlocks their access panel, and does not upload custom laws.
+You can redirect package-wrapped items using the delivery tagger and deliver them to a specific area.
+Items can be thrown down Z-levels to unsuspecting people below.
+Screwdrivers allow you to change the timers on a grenade to detonate at different times.
+Insulated Gloves can be redyed to another colour using a crayon in the washing machine.
+Budget Insulated Gloves will either reduce or multiply the amount of electrical shock damage you get.
+Leaving the Emitter on in the Engine Room is a unwise decision.
+Emagging the Fax Machine lets you send messages to syndicate contacts.
+Praying to the gods may grant you all sorts of boons, or just a cookie.
+The best judgements are always made on the decision of a magic 8-ball.
+It is good practice to remove a victim's headset and turn-off their suit sensors while they are still alive, to avoid medbay or security noticing.
```

Comment:
```suggestion
It is good practice to remove a victim's headset and turn off their suit sensors while they are still alive, to avoid medbay or security noticing.
```